misc/bazel/glibc_symbols_check.bzl

```diff
@@ -0,0 +1,19 @@
+load("@rules_python//python:defs.bzl", "py_test")
+
+def glibc_symbols_check(name, binary):
+    """
+    Checks that the supplied binary doesn't use symbols that are not available in older glibc versions.
+    """
+    # Note this accesses system binaries that are not declared anywhere,
```

Comment:
This can be removed iirc, as rules_python now ships a hermetic python interpreter (verification of that would be nice, though).